Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -493,7 +505,7 @@ protected function readCacheFile($cacheEntryPathAndFilename, $offset = null, $ma
             if ($data !== false) {
                 return $data;
             }
-            usleep(rand(10, 500));
+            usleep(random_int(10, 500));
```

Comment:
IMHO this is not a good idea because random_int is slower than rand and no cryptographically secure value is needed here.